nodes/src/nodes/multi_agent/blackboard.py

```
@@ -0,0 +1,102 @@
+    """
+
+    def __init__(self) -> None:  # noqa: D107
+        self._lock = threading.Lock()
+        self._store: Dict[str, Any] = {}
+        self._history: List[BlackboardEntry] = []
+
+    # ------------------------------------------------------------------
+    # Write
+    # ------------------------------------------------------------------
+
+    def write(self, agent_name: str, key: str, value: Any) -> None:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)